src/slider/tp-slider.ts

```diff
@@ -91,7 +98,126 @@ export class TPSliderElement extends HTMLElement {
 	 */
 	static get observedAttributes(): string[] {
 		// Observed attributes.
-		return [ 'current-slide', 'flexible-height', 'infinite', 'swipe', 'per-view', 'step' ];
+		return [ 'current-slide', 'flexible-height', 'infinite', 'swipe', 'per-view', 'step', 'draggable' ];
+	}
+
+	/**
+	 * Check and initialize draggable behavior.
+	 */
+	checkDraggable() {
```

Comment:
@AvinashVinod03 I wonder if draggable should be implied by `swipe` rather than creating a separate attribute? Have we evaluated the pros and cons?